summary/summary.py

```diff
@@ -36,7 +36,7 @@ def run(self):
         try:
             r = requests.get('https://apiv2.bitcoinaverage.com/convert/global'
                              '?from=BTC&to={}&amount=1'.format(plugin.currency))
-            plugin.fiat_per_btc = json.loads(r.content)['price']
+            plugin.fiat_per_btc = json.loads(r.content.decode())['price']
```

Comment:
```suggestion
            plugin.fiat_per_btc = json.loads(r.content.decode('UTF-8'))['price']
```

It's probably better to mention which encoding should be used here. UTF-8 is what we use everywhere else, and supports a superset of most other encodings. More importantly it also includes the EUR symbol :wink: